### ISSUE TITLE
Integrate boot starter with jhipster

### ISSUE THREAD

**Issue by ryandawsonuk** [Activiti/Activiti#1486]

**Comment by Salaboy** [comment]
This is for the monolith approach, but we need to go hard on the microservice way when they upgrade to spring boot 2.

**Comment by ryandawsonuk** [comment]
Yes this is for monolith. Do we want to create a bamboo job to publish the monolith integration to npm?

**Comment by Salaboy** [comment]
@ryandawsonuk maybe after DevCon. I've moved this to Jan
